### ISSUE TITLE
Add hostname, IP and port number in the summary output

### ISSUE THREAD

**Issue by BackSpace54** [pouriyajamshidi/tcping#63]
Hi,

**Is your feature request related to a problem? Please describe.**
No.

**Describe the solution you'd like**

- Your summary statistics is a very useful, is it possible to had some more info?

For e.g. have the ip address of the host name if it was resolved and the port number

$ tcping github.com 80
...
^C
--- github.com **(140.82.121.4)** TCPing statistics ---
2 probes transmitted **on port 80**, 2 received...
...

$ tcping example.com 25
...
^C
--- example.com **(2606:2800:220:1:248:1893:25c8:1946)** TCPing statistics ---
4 probes transmitted **on port 25**, 0 received...
...

$ tcping 140.82.121.4 21
...
^C
--- 140.82.121.4 TCPing statistics ---
6 probes transmitted **on port 21**, 0 received...
...

- <strike>Like classic ping command is it possible to have -4 and -6 switch to force IPv4 or IPv6 protocol?</strike>

Thanks.

**Comment by pouriyajamshidi** [comment]
Hi @BackSpace54,

Please help me understand the use case here. Port number and hostname with its corresponding IP address are already printed for each probe.

It sounds a bit redundant to me but I do not know your specific scenario.

**Comment by BackSpace54** [comment]
Hi,
thank for your reply.

Yep it's quite redundant with each probes, it could be useful to only transmit the summary and so to have the port and IP/host name (e.g. transmit summary to your host provider, to your IT team (to unlock a port blocked by firewall...), ... with all needed info).
If it's too complex to add no problem i could copy and paste all the info to the summary ;-)
I think all info in summary could be useful to others/users of tcping.

**Comment by pouriyajamshidi** [comment]
Hi,

It is not a complicated thing to add at all. My intention in life is to keep things simple :)

Fair enough. I will work on it myself if I do not get any pull requests. A bit busy these days at work.

May I ask you to open a separate issue for the `-4` `-6` flag?

It is better to keep issues of the same type in separation for easier tracking.

**Comment by BackSpace54** [comment]
Hi,
thanks for your time :+1: , as said it's to have all essentials info in the summary.

Ok i'll open a new issue for the -4 -6 switch.

**Comment by pouriyajamshidi** [comment]
Hi @BackSpace54 ,

Version 1.20.0 adds the requested info to the summary output.

**Comment by BackSpace54** [comment]
Hi,
thanks for adding all the infos 👍
